Userland/Libraries/LibMedia/CMakeLists.txt

```diff
@@ -15,3 +16,9 @@ set(SOURCES
 
 serenity_lib(LibMedia media)
 target_link_libraries(LibMedia PRIVATE LibCore LibIPC LibGfx LibThreading)
+
+find_package(PkgConfig)
+pkg_check_modules(LIBAV IMPORTED_TARGET libavcodec)
+
+target_include_directories(LibMedia PRIVATE ${LIBAV_INCLUDE_DIRS})
+target_link_libraries(LibMedia PRIVATE ${LIBAV_LIBRARIES})
```

Comment:
you should target_link_directories with the library dirs here as well. Assuming that the lib dir is in the default lib path is no bueno.